packages/wallet/backend/src/webMonetization/paymentPointer/model.ts

```diff
@@ -5,8 +5,11 @@ import { WMTransaction } from '../transaction/model'
 
 export class WMPaymentPointer extends PaymentPointerBaseModel {
   static tableName = 'wmPaymentPointers'
+  public isWM = true
 
   public balance!: bigint
+  public assetCode!: string
+  public assetScale!: number
```

Comment:
```suggestion
  static tableName = 'wmPaymentPointers'
  
  public isWM = true
  public balance!: bigint
  public assetCode!: string
  public assetScale!: number
```

Reply:
no longer needed, wmPaymentPointer model and table no longer exist